html-forms_personal-profile-form/index.html

```diff
@@ -10,6 +10,51 @@
     <link rel="stylesheet" href="./css/styles.css" />
   </head>
   <body>
-    <section></section>
+    <section>
+      <form>
+        <h3>
+          We want to get to know you! Please enter your personal information!
+        </h3>
+        <form>
```

Comment:
I only used one form tag and seperated the sections by using `<fieldset>` and `<legend>`

Reply:
Thank you, that was very helpful!